vsts/scripts/tagBaseImagesForRelease.sh

```diff
@@ -65,6 +65,7 @@ mkdir -p $artifactsDir/$imageName
 if [ "$imageName" == "node" ]
 then
   echo ""
+  retagImageWithStagingRepository node-runtimeimage-bases-bullseye.txt $imageName bullseye
   retagImageWithStagingRepository node-runtimeimage-bases-buster.txt $imageName buster
```

Comment:
Similar comment here: do we still need this step for buster if we're no longer creating buster images for Node?

Reply:
Thanks, will update it.